changelog.md

```diff
@@ -7,6 +7,34 @@ All notable end-user facing changes are documented in this file.
 *Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
 The idea is to track end-user facing changes as they occur.*
 
+### [2.0.0-beta2]
+
+- Known issues
+  - Code generation is too aggressive in generating serializers for most available types instead of just those that are directly used in grain methods. This causes excessive code being generated and compiled.
+
+- Breaking changes
+  - Remove `IGrainInvokeInterceptor` that got replaced with `IGrainCallFilter`
+  - Migrate more configuration settings to typed options
+  - Replace DeploymentId with ClusterId and collapse them where both are defined
+
+- Non-breaking improvements
+  - Better align silo hosting APIs with the future generic Microsoft.Extensions.Hosting.HostBuilder
+  - Multiple improvements to code generation
```

Comment:
Is it possible to link PR to those improvements, so that user will have more information about them? Our previous release notes mostly did that.